Description: need to manually link to libgl
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-03-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -2,8 +2,8 @@
 SHELL := /bin/sh
 CC    := gcc
 PKG_CONFIG:=pkg-config
-CFLAGS   += -Wall -pipe -std=c99 -pedantic $(shell $(PKG_CONFIG) glu sdl2 --cflags)
-LDFLAGS  += $(shell $(PKG_CONFIG) glu sdl2 --libs) -lm -pipe
+CFLAGS   += -Wall -pipe -std=c99 -pedantic $(shell $(PKG_CONFIG) glu gl sdl2 --cflags)
+LDFLAGS  += $(shell $(PKG_CONFIG) glu gl sdl2 --libs) -lm -pipe
 #variable de nettoyage
 RM_F := rm -f
 #variables d'instalation
